template/build/webpack.base.conf.js

```diff
@@ -31,76 +31,73 @@ module.exports = {
       'components': path.resolve(__dirname, '../src/components')
     }
   },
-  resolveLoader: {
-    fallback: [path.join(__dirname, '../node_modules')]
-  },
   module: {
+    rules: [
     {{#lint}}
-    preLoaders: [
       {
         test: /\.vue$/,
-        loader: 'eslint',
+        loader: 'eslint-loader',
+        enforce: 'pre',
         include: [
           path.join(projectRoot, 'src')
         ],
         exclude: /node_modules/
       },
       {
         test: /\.js$/,
-        loader: 'eslint',
+        loader: 'eslint-loader',
+        enforce: 'pre',
         include: [
           path.join(projectRoot, 'src')
         ],
         exclude: /node_modules/
-      }
-    ],
+      },
     {{/lint}}
-    loaders: [
       {
         test: /\.vue$/,
-        loader: 'vue'
+        loader: 'vue-loader'
       },
       {
         test: /\.js$/,
-        loader: 'babel',
+        loader: 'babel-loader',
         include: [
           path.join(projectRoot, 'src')
         ],
         exclude: /node_modules/
```

Comment:
I assume it was meant as a safeguard in case people add other paths to the included paths array.

But I think we could remove it to trim some fat here.